packages/web-worker/src/endpoint.ts

```diff
@@ -146,6 +154,14 @@ export function createEndpoint<T>(
       makeCallable(port, () => newValue);
       functionStore.set(newValue, [id, port]);
     },
+    terminate() {
```

Comment:
We talked about `self.close()` being available from the worker scope. Looking at the latest [`WorkerGlobalScope.close()` docs](https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/close) , this is a deprecated API .